client/setup.py

```diff
@@ -23,8 +23,9 @@
         "protobuf>=3.20.1",
         # gRPC,
         "grpcio>=1.46.1",
-        # PyYML package to parse yaml config files.
+        # Used to marshal config file data.
```

Comment:
move to the next line?

Reply:
I put it above `PyYML` because `PyYML` and `marshmallow` are both doing the work. I should probably change it to something like "Used to read and validate config file data".